VnE/Source/Extensions/ChemPack/instruments.py

```
@@ -95,25 +100,84 @@ def show(self):
+                c = c*w
+                c = m_inv @ c
+                d = c[:-1,0] - c_old
+                coords = np.squeeze(d).astype(np.float32)
+            else:
+                continue
+            if p.label_shift is not None:
+                p.label_shift += coords
+            else:
+                p.addProperty('label_shift', coords)
+        self.opengl_widget.update()
         ...
```

Ruff (0.14.14)

[warning] Unnecessary `...` literal

Remove unnecessary `...`

(PIE790)